modules/ROOT/pages/app-dashboards.adoc

```diff
@@ -174,6 +174,14 @@ image::dashboard-built-in-infrastructure.png[Example: Infrastructure Dashboard]
 The *Custom metrics* tab displays only if you have a Titanium subscription and select an app running on a supported version of Mule. See xref:anypoint-custom-metrics-connector.adoc[Anypoint Custom Metrics Connector].
 |===
 
+=== Charts with Endpoints
+
+Charts that contain endpoint metrics sort the available endpoints in a menu.
+The requested time period for the results shown in the endpoint charts can include multiple grouping intervals because the data is processed in intervals of time.
+As a result, a given table usually shows top occurring endpoints and, depending on the number of endpoints, a group called *Other* that shows an aggregation of endpoints that do not occur as often for that time period.The tables show approximate counts for the most commonly occurring endpoints when the *Other* grouping appears.
+
+For example, when an endpoint occurs more often than other endpoints in one interval but occurs less often in another interval (thus sorting it into the *Other* group for that interval), the resulting table displays lower than actual values for that endpoint, since the metrics that do not appear are grouped into the *Other* group.
+
```

Comment:
How Endpoint Use Metrics Are Gathered

When you request endpoint use metrics for a particular time period, the results can be a little misleading. This is because:

- Time periods comprise multiple intervals.
<illustrate?>
- At each interval, endpoint use is ranked either individually by frequency, with the least frequently used endpoints grouped as *Other*
<illustrate?>
- For any one time period, an endpoint might be counted independently in some intervals and as part of a group
   in others.
<illustrate?>
- Group statistics hide the constituent endpoints' use frequency.
<illustrate?>